src/main/java/net/dv8tion/jda/core/handle/MessageDeleteHandler.java

```diff
@@ -58,8 +58,9 @@ protected Long handleInternally(JSONObject content)
         }
         if (channel == null)
         {
-            api.getEventCache().cache(EventCache.Type.CHANNEL, channelId, () -> handle(responseNumber, allContent));
-            EventCache.LOG.debug("Got message delete for a channel/group that is not yet cached. ChannelId: " + channelId);
+//            api.getEventCache().cache(EventCache.Type.CHANNEL, channelId, () -> handle(responseNumber, allContent));
+//            EventCache.LOG.debug("Got message delete for a channel/group that is not yet cached. ChannelId: " + channelId);
+            // Don't cache message events for channels unless they are MESSAGE_CREATE
```

Comment:
Added reference to the specific comment in the Github issue as to *why* we don't cache anymore